[Document] Introduces the document of the command line interface (#204)
flink-extended/ai-flow: docs/content/cli/notification.md

# Notification

## Command Line Interface

> Notification has a very rich command line interface that allows for many types of operation on a Events, starting services, and supporting development and testing.

Reviewer: ```suggestion
Notification has a very rich command-line interface that supports many types of operations on Events, starting services and testing.
```

```

##### Positional Arguments

> key

Key of the event.

##### Named Arguments

> -s, --server-uri

The uri of notification server.

> -n, --namespace

Namespace of the event. If not set, all namespaces would be handled.

> --begin-version

Begin version of the event.  
Default:  0.

> --event_type

Type of the event. If not set, all types would be handled.

> --begin-time

Begin datetime of the event to listen, formatted in ISO 8601.
Default: `datetime.now().isoformat()`.

> --sender

Sender of the event.

##### send

Sends an event.

```
notification event send [-h] key value [-s SERVER_URI] [-n NAMESPACE] [--event_type EVENT_TYPE] [--context CONTEXT] [--sender SENDER]
```

##### Positional Arguments

> key

Key of the event.

> value

Value of the event.

##### Named Arguments

> -s, --server-uri

The uri of notification server.

> -n, --namespace

Namespace of the event. If not set, all namespaces would be handled.

> --event_type

Type of the event. If not set, all types would be handled.

> --context

Context of the event.

> --sender

Sender of the event.

(notification-cli-config)=

#### config

> Manage configuration.

Reviewer: ```suggestion
Manages configuration.
```

```
notification config [-h] COMMAND ... 
```

#### Positional Arguments

> COMMAND

Possible choices: get-value, init, list.

#### Sub-commands

##### get-value

Gets the option value of the configuration.

```
notification config get-value [-h] option
```
 
##### Positional Arguments

> option

The option name of the configuration.

##### init

Initializes the default configuration.

```
notification config init [-h]
```

##### list

> List all options of the configuration.

Reviewer: ```suggestion
Lists all options of the configuration.
```
